### PR TITLE
fix up ROS network. Add nav_integration repo module. fix SBG mounting

### PR DESCRIPTION
**Pull request by b1n-ch1kn**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimisation

## [required] Description

Fix up ROS networking for new jetson LAN connection.
Add nav_integration repo module - future work to copy these in and deprecate development in that repo.
Fix SBG mounting, standard serial dev rules.

## [required] Documentation

https://docs.google.com/document/d/19m3uLLtovWiaIzsZ1k3DOFrNCYlGSvK4GUioXfJtE8g/edit

### [optional] Usability concerns or breaking changes?

Requires the container to be rebuilt. No removal of reqs etc

### CODE REVIEW

File: .gitmodules
Comment (Grant-ed) [inline .gitmodules:1]: should the sbg ros2 driver be added as a submodule as well?

File: docker/Dockerfile.base
Comment (Grant-ed) [inline docker/Dockerfile.base:1]: Again, should the sbg_ros2_driver folder  be added here?

File: docker/roscube_startup.sh
Comment (Grant-ed) [inline docker/roscube_startup.sh:1]: Not sure if we should clean some of the comments up here, or if we want the comments to revert to it in future, we should explicitly say that somewhere (and document that this is temporary).

File: .gitmodules
Comment (b1n-ch1kn) [reply .gitmodules:1]: wasnt intended to be, as it get cloned at a specific commit for the PC. Ideally i'd like to move away from the SBG driver being a requirement for the sim too, and instead better simulate all of our devices / use standard ROS msgs over SBG custom msgs

File: docker/Dockerfile.base
Comment (b1n-ch1kn) [reply docker/Dockerfile.base:1]: No and I'm not too sure if that line adding `QUTMS_Nav_Integration` even works there - thats linking volumes so you can change it locally on the host env and the docker container can see changes. Unlike cloning directly into the container which the host cant see

File: docker/roscube_startup.sh
Comment (b1n-ch1kn) [reply docker/roscube_startup.sh:1]: Given the `startup.service` systemctl file cant be committed, this was the closest thing to it. When we changed Jetsons we had no idea what went into the .service file

**Review by Grant-ed — APPROVED** [review]

**Pull request merged**